examples/vue2/rsbuild.config.ts

```diff
@@ -3,4 +3,10 @@ import { pluginVue2 } from '@rsbuild/plugin-vue2';
 
 export default defineConfig({
   plugins: [pluginVue2()],
+  server: {
+    publicDir: {
+      name: 'public1',
```

Comment:
is this a temp code😄

Reply:
😅😅sorry, this is my debug code, i has removed.